src/imio/news/core/rest/search/endpoint.py

```
@@ -1,80 +1,136 @@
+    enable_prefix = True
+    if isinstance(prefix_param, str):
+        lowered = prefix_param.strip().lower()
+        if lowered in {"0", "false", "no"}:
+            enable_prefix = False
+    st = query.get("SearchableText")
+    if enable_prefix and isinstance(st, str) and st and not st.endswith("*"):
+        query["SearchableText"] = f"{st}*"
+    return unflatten_dotted_dict(query)
+
+
+def _cachekey_by_entity_uid(method, self, entity_uid_key):
```

Ruff (0.15.2)

[warning] Unused function argument: `method`

(ARG001)